conf.py

```diff
@@ -64,10 +64,10 @@
     'sphinx.ext.mathjax',
     'sphinx.ext.extlinks',
     'nbsphinx',
+    "qiskit_sphinx_theme",
```

Comment:
Is this necessary now because the them ships with custom directives and other things?

Reply:
In 1.11, we have qiskit_sphinx_theme automatically activate jQuery. But you have to put qiskit_sphinx_theme in the extensions list to have that work.

In 1.12+, it's good to also have qiskit_sphinx_theme in the extensions list because it activates the custom directives, Translations feature, and Previous Releases feature. So, we now expect all projects to have this config when they use our theme. I've finished updating every project to do that.

Reply:
Thanks for the detailed answer. That's very helpful!